### DISCUSSION TITLE
[小说]之前写的咖啡馆恐怖小说，看看能不能在这发

### DISCUSSION THREAD

**Discussion by aaa00073** [share121/inter-knot#413] [其他]
第一章
    深夜咖啡馆。

　　“所以，你平常上班的地方就是医院停尸房，对吗？”

　　“嗯。”

　　面对陈往的询问，红衣女人轻轻点了点头。

　　小铁勺在杯中轻轻搅动，不时撞在杯壁上，发出叮叮的声音。

　　女人右手撑着脑袋，耷拉着眼皮，没精打采的说道：“但是我已经很久没去上班了。”

　　“为什么？”陈往端起咖啡杯喝了一口。

　　“停尸房出了点事，我不敢再去了。”

　　女人轻轻端起咖啡抿了一口，随手拿起一张纸，擦了擦脖子。

　　“说说看。”

　　放下杯子，女人讲述道：“那天晚上，我本来要去给冷冻柜做日常的消毒和清洁工作，但是在我做完了清洁准备走的时候，我突然听见某个消毒柜发出了声响。”

　　陈往身体前倾，问道：“是什么样声响？”

　　女人轻轻蹙眉，回忆道：“那种声响很奇怪，我也难以形容。”

　　“就像是有什么东西撞到了冷冻柜，我不确定是哪个冷冻柜发出的声音。”

　　“我很害怕，我的腿像是不受控制似得僵在原地，我花了好久时间，终于让脚挪动了一点点。”

　　“然后，那声响又出现了。”

　　女人又停顿了，眼睛不知道看向何处。

　　陈往表情有些僵硬，感觉坐得很不自在。

　　“那……”

　　“那之后呢，你跑出去了吗？”

　　女人摇了摇头。

　　“我全身都瘫软了，我发现自己太高估自己了，我根本跑不动，只觉得双腿发麻。”

　　“但是我听到了那声音来自哪一个柜子。”

　　“我花了很久，思考要不要把尸体拉出来查看一下情况。”

　　“最终我还是这样做了。”

　　陈往：“所以，柜子当中有什么？”

　　女人再次轻抿咖啡，白色的嘴唇被咖啡染上浅棕色。

　　“我将柜子拉出来了，但出人意料的是……”

　　“一切都很正常，尸体放的端端正正的，唯一不同的地方只有，那具尸体没有脑袋，是一具无头尸体。”

　　“我见过各种尸体了，所以这并没有什么特别的，我检查了一下就把尸体推回去了。”

　　陈往问道：“所以，你还是没有找到响声的来源吗？”

　　女人捏了捏嗓子，回答道：

　　“后来外面忽然响起了敲门声，然后是我同事小玲的声音。”

　　“她在门外问我怎么了，我就把刚才发生的事告诉了她。”

　　“小玲说，这里可能有老鼠，可能在哪个角落弄出了点动静。”

　　“我其实不太怕老鼠，听她这么说，我反倒是送了一口气。”

　　“我回她，说我想试试能不能把老鼠找出来。”

　　“小玲说，她可以帮我找找。”

　　“我说可以，进来吧。”

　　“然后小玲就推开门进来了。”

　　“然后，我就看见，”

　　“小玲她……”

　　“没有脑袋……”

　　“……”陈往。

　　正在用勺子搅咖啡的陈往，动作戛然而止。

　　但是，

　　女人这次，却一点也没有停顿。

　　“当时我的感觉就像，被勒住了喉咙，完全停止住了呼吸，大脑一片空白。”

　　“我再也动不了了，我很想就这么直接晕过去。”

　　“但是我并没有，我眼睁睁地看着小玲走过来，拉起我的手，还跟我闲聊着。”

　　“她是怎么说话的？她嘴长在哪里的？她为什么还活着。”

　　“我的大脑已经乱成了一锅粥。”

　　“我难以再控制身体，感觉全身失去知觉，摔倒了下去。”

　　“很痛。”

　　“摔到我的脑袋，很痛很痛。”

　　“我那一刻心想，这一摔，摔晕过去该多好。”

　　“但是，”

　　“直到我感觉脖颈凉嗖嗖的。”

　　“你说，我的脑袋是不是也掉了？”

　　“……”

　　陈往终于感到全身僵硬，如坠冰窟。

　　他的眼睛，一直都死死盯着咖啡。

　　端起咖啡，强行镇定的喝了一口，陈往说了句：“这个玩笑挺有意思的。”

　　实际上，他装作镇定很久了。

　　他早就双腿麻木，后背冒出冷汗了。

　　但是，他一直在装作平静。

　　平静的接着这女人的话茬。

　　就像两个普通人那样闲聊。

　　不管是对什么内容，陈往都尽量做出不那么大的反应。

　　没人知道，陈往已经在这里坐了多久了。

　　但是陈往，注意到一件事，已经很久很久了。

　　这个女人，

　　从头到尾，

　　一直用手撑着脑袋。

　　每次喝一口，女人都会用纸擦一下脖子。

　　因为，

　　那里有一条缝，咖啡会从那里渗出来……

　　陈往强行镇定下来。

　　九个小时。

　　已经九个小时了。

　　陈往坐在这里听这个“女人”讲故事，已经九个小时。

　　停尸房里掉脑袋的故事，陈往已经听了几十遍。

　　陈往后悔走进这家深夜咖啡馆了。

　　更后悔自己受其邀请，坐在她的对面。

　　从那一刻开始，咖啡馆外的天空就没亮起过。

　　外面的雨就没停过。

　　一开始，陈往很庆幸，自己在发现女人身上的异常之后，还能保持镇定。

　　但现在，陈往很后悔。

　　要是当时自己没有沉住气，大叫一声，或者露出害怕的表情，恐怕早就结束这一切。

　　现在在这里渡过九个小时，陈往比坐牢还痛苦。

　　简直就是在受刑。

　　陈往现在就像那个故事里的女人一样，多么希望出现什么变故，把这一切都打断。

　　不管自己死没死，赶紧结束这种恐惧吧，陈往一秒也不想待下去了。

　　他端起咖啡，再次喝了一口。

　　女人用空出来的手掀开耷拉着的眼皮，将快要脱落出来眼球按了回去，然后又摸了摸。

　　冷静，冷静。

　　陈往盯着女人的脖颈。

　　他忽然发现，断口好像有点移位了。

　　发现这一点之后，陈往再也移不开眼睛，视线牢牢钉在那里。

　　脑袋的移位不多，却很明显，这让陈往很难受。

　　他脑海里甚至萌生出一种想法。

　　伸手将其扶正。

　　可是他不敢，他只会儿不停喝手中从来不见少的咖啡。

　　要不要……

　　要不要提醒她？

　　可是，如果提醒她，就相当于捅破了那层窗户纸。

　　会发生什么，他也无法确定。

　　可是就在这样的情况下，陈往发现，

　　那偏移竟然在逐渐变大。

　　完了，该不会掉下来吧？

　　陈往无法想象，

　　但陈往脑海中，一直构建着脑袋砰地一声砸在桌子上的声音。

　　求求你了，用两个手一起撑住吧！

　　真的要掉下来了！

　　喝咖啡。

　　陈往：“后来呢？怎么样了？”

　　女人：“后来，我离开了停尸房，去了一家咖啡馆……”

　　
![IMG_20240724_220546](https://github.com/user-attachments/assets/17ea4bcb-3be6-4bf1-96b7-e1a892365f68)

**Comment by aaa00073** [comment]
内投没过的稿子，一共只有三章，如果大家觉得不错，我再把后两章发出来

**Comment by 15935784260** [comment]
还蛮有意思的！❤️❤️

**Comment by haihe8177** [comment]
爱看，求后续❤️

**Comment by Hexiaoling457** [comment]
好看，爱看！

**Comment by LadySunBringer** [comment]
hentai群主！才......才不给你评论呢！

**Comment by XiaoFffeng** [comment]
怎么哪里都有傲娇（

**Comment by justafish8787** [comment]
好看(´▽`ʃ♡ƪ)

**Comment by Qiling-XwX** [comment]
求后续！

**Reply by justafish8787** [reply]
已经发了，去他的首页看看

**Comment by lllliufeng** [comment]
吹羹.jpg

**Comment by Signedinn** [comment]
写的挺好
